drivers/sensor/nordic/qdec_nrfx/qdec_nrfx.c

```diff
@@ -45,7 +47,7 @@ static void accumulate(struct qdec_nrfx_data *data, int32_t acc)
 	unsigned int key = irq_lock();
 
 	bool overflow = ((acc > 0) && (ACC_MAX - acc < data->acc)) ||
-			((acc < 0) && (ACC_MIN - acc > data->acc));
+					((acc < 0) && (ACC_MIN - acc > data->acc));
```

Comment:
This change is undesirable. Please change the tab size in your [settings](https://github.com/settings/appearance) to 8 (to align with Zephyr [coding style](https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style)) and see how the above is rendered then.
After correcting the tab size, look also here: https://github.com/zephyrproject-rtos/zephyr/blob/f92c8195382bd3e264d8514459dfbe16cf3ad1cb/tests/boards/nrf/qdec/src/main.c#L322-L323

Reply:
Thank you for the tips, I actually had a misalignment between code editor and GH. Solved now